format.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
+echo "Running mypy"
+mypy dags/benk
```

Comment:
mypy format niks volgens mij?

Reply:
Het voegt whitelines toe in import statements.

Reply:
apart, daar is isort voor.. maar het zal